### PR TITLE
Update to v1 of digime-sdk-js

### PR DESCRIPTION
**Pull request by rickydigime**
This updates the example application to utilise the latest v1.0.0 of the
SDK. Also updated is the documentation.

### CODE REVIEW

**Comment by rickydigime** [comment]
From Martin:

image

File: README.md [hunk 1]
```diff
@@ -38,7 +44,21 @@ In order to run this example app, you'll need to perform some preparation steps:
...
     - The second URL is our best guess as to what your local network IP is. You should be able to open the example app from your phone with it, assuming that both machines are on the same network.
 
-## Additional information
+## To run the private share example application
+
+In order to run this example app, you'll need to perform some preparation steps. From digi.me, you'll need to obtain the following:
+* Contract ID
+* Application ID
+* Private key (PKCS1 PEM)
+* Depending on which private sharing mode you run, you may need a valid Digi.me client application ([See below for more info](#setting-up-the-digi.me-library))
```
Comment (olileung) [inline README.md]: digi.me should be the lower case as it's used in the middle of a sentence

File: examples/private-sharing/index.js [hunk 2]
```diff
@@ -87,21 +88,21 @@ app.get("/return", (req, res) => {
...
+    if (result !== "SUCCESS") {
         res.render("pages/error");
         return;
     }
 
     // If we get do get the information that the private sharing request was APPROVED,
     // the data is ready to be retrieved and consumed!
 
-    // Here, we're using `getDataForSession` to retrieve the data from Digi.me API,
+    // Here, we're using `getSessionData` to retrieve the data from Digi.me API,
```
Comment (olileung) [inline examples/private-sharing/index.js]: digi.me should be lowercase in this instance

File: examples/private-sharing/index.js [hunk 3]
```diff
@@ -12,11 +12,12 @@ app.set("views", __dirname + "/views");
...
-// const { establishSession, getWebURL, getDataForSession, getAppURL } = createSDK({ host: "api.digi.me" });
+// const { init } = require("@digime/digime-js-sdk");
+// const { establishSession, getGuestAuthorizeUrl, getSessionData, getAuthorizeUrl } = init({ baseUrl: "https://api.development.devdigi.me/v1.4" });
+
 
 // Since we do not need to specify different initialization options we will import the functions directly:
-const { establishSession, getWebURL, getDataForSession, getAppURL } = require("@digime/digime-js-sdk");
+const { establishSession, getGuestAuthorizeUrl, getSessionData, getAuthorizeUrl } = require("@digime/digime-js-sdk");
 
 // Options that we will pass to the Digi.me SDK
```
Comment (olileung) [inline examples/private-sharing/index.js]: digi.me should be lowercase in this instance

**Pull request merged**